zest/releaser/tests/pyproject-toml.txt

```diff
@@ -88,20 +88,23 @@ known on PyPI:
     Our reply: y
     <BLANKLINE>
     Question: Check out the tag
-        (for tweaks or pypi/distutils server upload) (y/N)?
+        (for tweaks or pypi/distutils server upload) (Y/n)?
```

Comment:
The default answer changed. Is that right? On line 75-79 it suggests that the answer should be "no".

- It could be that we were abusing the pyproject.toml test for a test that wasn't quite related.
- It can also be that the behaviour changed somewhere in a way that's not right.
- Or the test text should be changed.

I'm not sure right now which one it is.

Reply:
The pyproject-toml.txt test was basically a copy of the functional-git.txt test, with the difference that packages containing only a pyproject.toml couldn't be built, so they weren't seen as a pip package and hence weren't uploaded to pip. With the new build system, they can be built, so the default answer has been changed to upload to pip just like with a setup.py.

Reply:
Ok